src/main/resources/views/CommercialVehicles/MSVA30.hbs

```diff
@@ -76,12 +75,12 @@
             {{#if msvaData.requiredStandards}}
                 <ul class="required-standard-list">
                     {{#each msvaData.requiredStandards}}
-                        <li>
+                        <li class= "required-standard-list-item">
                             <div>
                                 <p><span class="required-standard-section">{{this.sectionNumber}} - {{this.sectionDescription}}</span></p>
                             </div>
                             <div>
-                                <p><span> RS{{this.rsNumber}}: {{this.requiredStandard}} </span></p>
+                                <p><span> RS{{this.rsNumber}}: {{this.requiredStandard}} {{#if this.prs}} (PRS) {{/if}}</span></p>
```

Comment:
Can you please add a unit test validating that `(PRS)` is displayed agains the required standard in the event the value is true.